src/seeders/20210127161802-add-goals.js

```diff
@@ -12,82 +12,99 @@ const bulletedGoal = `
+        createdAt: now,
+        updatedAt: now,
+        templateNameModifiedAt: now,
+        creationMethod: 'Automatic',
+      },
+      {
+        id: 2,
+        hash: queryInterface.sequelize.fn('md5', queryInterface.sequelize.fn('TRIM', 'Recipient supports and sustains comprehensive, integrated and systemic SR, PFCE, and PD processes and services.')),
+        templateName: 'Recipient supports and sustains comprehensive, integrated and systemic SR, PFCE, and PD processes and services.',
+        createdAt: now,
+        updatedAt: now,
+        templateNameModifiedAt: now,
+        creationMethod: 'Automatic',
+      },
+      {
+        id: 3,
+        hash: queryInterface.sequelize.fn('md5', queryInterface.sequelize.fn('TRIM', bulletedGoal)),
+        templateName: bulletedGoal,
+        createdAt: now,
+        updatedAt: now,
+        templateNameModifiedAt: now,
+        creationMethod: 'Automatic',
+      },
+      {
+        id: 4,
+        hash: queryInterface.sequelize.fn('md5', queryInterface.sequelize.fn('TRIM', longGoal)),
+        templateName: longGoal,
+        createdAt: now,
+        updatedAt: now,
+        templateNameModifiedAt: now,
+        creationMethod: 'Automatic',
+      },
+    ];
+
+    const goals = [
+      {
+        id: 1,
+        name: 'Identify strategies to support Professional Development with an emphasis on Staff Wellness and Social Emotional Development.',
+        status: 'Not Started',
+        createdAt: now,
+        updatedAt: now,
+        goalTemplateId: 1,
+        grantId: 1,
+        onApprovedAR: false,
+      },
+      {
+        id: 2,
+        name: 'Recipient supports and sustains comprehensive, integrated and systemic SR, PFCE, and PD processes and services.',
+        status: 'Not Started',
+        createdAt: now,
+        updatedAt: now,
+        goalTemplateId: 2,
+        grantId: 1,
+        onApprovedAR: false,
+      },
+      {
+        id: 3,
+        name: bulletedGoal,
+        status: 'Not Started',
+        createdAt: now,
+        updatedAt: now,
+        goalTemplateId: 3,
+        grantId: 1,
+        onApprovedAR: false,
+      },
+      {
+        id: 4,
+        name: longGoal,
+        status: 'Not Started',
+        createdAt: now,
+        updatedAt: now,
+        goalTemplateId: 4,
+        grantId: 2,
+        onApprovedAR: false,
+      },
+    ];
+
+    await queryInterface.bulkInsert('GoalTemplates', goalTemplates, { validate: true, individualHooks: true });
+    await queryInterface.bulkInsert('Goals', goals, { validate: true, individualHooks: true });
+
+    await queryInterface.sequelize.query(`ALTER SEQUENCE "GoalTemplates_id_seq" RESTART WITH ${goalTemplates[goalTemplates.length - 1].id + 1};`);
+    await queryInterface.sequelize.query(`ALTER SEQUENCE "Goals_id_seq" RESTART WITH ${goals[goals.length - 1].id + 1};`);
```

Comment:
Seems weird that we have to do this. I assume there are bugs without these lines. Do you know why we have to restart the goal and goal template id sequences?